tests/test_select.py

```diff
@@ -60,22 +61,41 @@ def test_select(
 @pytest.mark.parametrize("struct", [False, True])
 @pytest.mark.parametrize("default", [10, None])
 @pytest.mark.parametrize(
-    "cfg, key",
+    ["cfg", "key", "exc_no_default"],
```

Comment:
Use tuple for consistency with the rest of the code.
```suggestion
    ("cfg", "key", "exc_no_default"),
```

Reply:
Done in 509bda5 (also on another instance in the same file)